ocelot/urls.py

```diff
@@ -52,6 +53,7 @@
     path('welcome', welcome, name='welcome-view'),
     # evaluation app
     path('compare/<sub_a_id>/<sub_b_id>', compare, name='compare-view'),
+    path('compare', compare, name='compare-view'),
```

Comment:
@snukky: this really is the `submit-rank-for-comparison` view... The name is confusing.